admin/translations/en.json

```diff
@@ -13,6 +13,9 @@
       "login-user-blocked-error": "Your account has reached the maximum number of failed login attempts and has been temporarily blocked. Please '<'a href=\"{url}\"'>'contact us'</a>' to unblock it.",
       "login-user-temporary-blocked-error": "You have reached too many failed login attempts. Please try again later or '<'a href=\"{url}\"'>'reset your password here'</a>'."
     },
+    "forms": {
+      "mandatory-fields": "Les champs marqués de '<abbr title=\"obligatoire\" class=\"mandatory-mark\">'*'</abbr>' sont obligatoires."
```

Comment:
```suggestion
      "mandatory-fields": "Fields tagged with '<abbr title=\"mandatory\" class=\"mandatory-mark\">'*'</abbr>' are mandatories."
```

Reply:
En règle général, on ne traduit pas (les devs) nous même les phrases/mots pour le front, on passe par une équipe de traduction (ou quand l'équipe de trad est pas dispo, un.e PO qui le gère).
Et dans le cas de Pix Admin, on ne fait jamais (ou très rarement) de la traduction car c'est un outil interne qui n'a pas vocation à passer à l'international dans l'absolu. Donc il est un peu oublié et par habitude on duplique le français dans le en.json.

Sauf que dans ce cas précis, on a déjà la traduction dans le code (coté pix app) car ce wording est censé être iso entre les applications. Je ne vais donc pas prendre ta suggestion donc merci de l'avoir noté. Je met la bonne traduction à la place :)

Reply:
mais du coup on reprends pas la trad de pix app? Iic c'est en francais

Reply:
si si en conclusion je vais mettre la trad anglais existante dans pix app